helm/values.yaml

```diff
@@ -45,7 +45,7 @@ mcLabelsAndAnnotations:
 
 image:
   repository: 3d-gateway
-  tag: latest
+  tag: v2.7.4
```

Comment:
I think we should delete it as we take by default the chart appversion that is changed via the release process

<img width="772" height="579" alt="Image" src="https://github.com/user-attachments/assets/31b6dcb1-ffde-4c5f-bd7c-94d6adb9eb12" />